docs/pages/attributes.rst

```diff
@@ -0,0 +1,215 @@
+:ref-prefix:
+    habitat_sim.sim
+    habitat_sim.attributes
+
+Using JSON Files to configure Attributes
+########################################
+
+:summary: Attributes templates are used in Habitat to configure the simulation world and specify characteristics for scenes and objects. This document describes the appropriate tags to be used when authoring JSON files to properly customize these various Attributes templates.
+
+.. contents::
+    :class: m-block m-default
+
+Attributes templates provide a mechanism by which the various constructions in Habitat can be constructed with user-specified characteristics, either at program start or on the fly.
+
+`Physics Manager Attributes`_
+=============================
+Physics Manager Attributes templates describe quantities pertinent to building the simulation world.  The descriptor JSON files used for these attributes should be named in the following format : 
+
+ 	<worldname>.phys_scene_config.json
+
+
+Below are the supported JSON tags for Physics Manager Attributes templates, and their meanings.
+
+"physics simulator"
+	- string
+	- What physics engine should be used for dynamics simulation.  Currently supports "bullet" for Bullet physics simulation, and "none", meaning kinematic motion is to be used.
+	
+"gravity"  
+	- 3-vector
+	- Gravity to use in scene for physical modeling.
+
+"timestep" 
+	- double
+	- The timestep to use for forward simulation.
+ 
+"friction coefficient"
+	- double
+	- The coefficient of friction.
+ 
+"restitution coefficient"
+	- double
+	- The coefficient of restitution.
+ 
+"rigid object paths"
+	- list of strings
+	- A list of locations to query for supported object files that should be available to be loaded in the world.
+
+
+`Scene Attributes`_
+===================
+Scene Attributes templates hold relevant information describing a scene's mesh, geometry and physical properties.  These files should be named in the format : 
+
+ 	<scenename>.scene_config.json
+	
+Scene Mesh Handles And Types
+----------------------------	
+
+Below are the handles and descriptors for various mesh assets used by a scene.
+	
+Mesh types are included to support future format flexibility. In the JSON they are limited to string keys used by :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap`, which map to values in :ref:`AssetType`.  Any mesh type specification used in JSON but not found in :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap` results in that mesh type being set to :ref:`AssetType::UNKNOWN`.  JSON-Specified mesh types will override any default settings, such as those that might be derived from file name extensions.
+
+"render mesh" 
+	- string
+	- The name of the file describing the render mesh to be used by the scene. 
```

Comment:
We talked about terminology around "scene" and "background/static" yesterday. I'd still like us to move to terminology where the "scene" is everything we simulate and render, and then we have a separate term like "background" to refer to this static object/mesh.

Reply:
It might be as simple as renaming this field to "background render mesh" or "static render mesh".